users/templates/signup.html

```diff
@@ -1,24 +1,43 @@
 {% extends 'base.html' %}
 
 {% load i18n %}
-{% load rest_framework %}
 
 {% block content %}
 <h2>{% trans 'Sign up' %}</h2>
 
 <div class="modal-dialog" style="margin-bottom:0">
-  <form action="{% url 'signup' %}" method="post">
-      {% csrf_token %}
-      {% render_form serializer template_pack='rest_framework/vertical' %}
-
-      {% if non_field_errors %}
-      <div class="form-group has-error">
-          {% for error in non_field_errors %}
-            <span class="help-block">{{ error }}</span><br>
-          {% endfor %}
+    <form action="{% url 'signup' %}" method="post">
+        {% csrf_token %}
+        <div class="fieldWrapper">
+            <label for="{{ form.email.label }}">Email:</label>
+                {{ form.email }}
+                {% if form.non_field_errors or form.email.errors %}
+                        <div class="alert alert-danger">
```

Comment:
Here you are wrapping whole errors list in div, but here https://github.com/Code-Poets/sheetstorm/pull/145/files#diff-0d040cecf685f46b2f769b9db182b639R34 you are wrapping each error separately. Please unify the way you do it and extract this to separate partial template and include it https://docs.djangoproject.com/en/2.2/ref/templates/builtins/#include.

Reply:
done

Reply:
Not exactly what I have meant, please create one universal mini-template for this (with generic name) and use it for each field in a form. Those included templates can take parameters, so you can do something like `{% include 'partials/errors_in_field.html'  with erorrs=form.password2.errors %}`

Reply:
Done

Reply:
Almost good, but why you are iterating twice over the same list, one inside another? :) I think once should be enough, or am I missing something?

Reply:
done